packages/react-async/test-d/types.test-d.ts

```diff
@@ -0,0 +1,5 @@
+import {expectAssignable} from 'tsd';
```

Comment:
We've  proved that the the loom integration works across multiple packages, but this test itself isn't that useful. Let's remove this